spec/requests/atmosphere/api/user_keys_spec.rb

```diff
@@ -184,12 +181,8 @@
         }
         post api("/user_keys", user), invalid_user_key_req
         expect(response.status).to eq 422
-        expect(json_response['details']).to eq({
-          'public_key' => [
-            'bad type of key (only ssh-rsa is allowed)',
-            'is invalid'
-          ]
-        })
+        expect(json_response['details']['public_key']).
+          to include('bad type of key (only ssh-rsa is allowed)')
```

Comment:
`include` checks only for selected element in array (https://www.relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/include-matcher). The problem I find here is that "is invalid" was sometime added, other time doesn't into error list and "is invalid" is not a case in this test only information about supported key type.